docs/source/models.rst

```diff
@@ -98,6 +98,8 @@ Identifying strings in images
 All text recognition models above have been evaluated using both the training and evaluation sets of FUNSD and CORD (cf. :ref:`datasets`).
 Explanations about the metrics being used are available in :ref:`metrics`.
 
+All these recognition models are trained with our french vocab (cf. :ref:`vocabs`).
```

Comment:
If I'm not mistaking, for now, we're using a custom alphabet (not in the same order as the french's one?)

Reply:
yes, do I need to change the order in the doc ?

Reply:
yes, do I need to change the order in the docs ?

Reply:
No that's alright, we're gonna change the model soon anyway!